src/fake_pokemon.js

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable */
+const pokemon = {
+  id: 35,
+  name: 'clefairy',
+  base_experience: 113,
+  height: 6,
+  is_default: true,
+  order: 56,
+  weight: 75,
+  abilities: [
```

Comment:
While I do understand this is a mock object to test the functionality of the popup window, let's keep in mind that including an object of this magnitude in our final code may affect our site's performance. There is nothing to change here, just a comment of something to be attentive to. I added just a couple of lines for reference but I hope you understand what I mean.